src/engine/BMInterfaceGame.php

```diff
@@ -49,15 +48,29 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                $currentPlayerId = $playerIdArray[0];
```

Comment:
BTW, this line is here to maintain current behaviour with unit tests and external game creation via Python, etc.

I'm thinking that it might actually be better to throw an exception if the $currentPlayerId is NULL, but that may break stuff with your current model of game creation. Let me know what you think.